src/js/api/order.js

```diff
@@ -549,6 +547,165 @@ ripe.Ripe.prototype.logOrderP = function(number, options) {
+ *
+ * If the option `states` is set to true, it returns the general chat
+ * lines merged with each state's information summarized (e.g. number of
+ * messages for that state and authors) taking into account the ordering
+ * of events, meaning state updates can appear between chat lines.
+ *
+ * @param {Number} number The number of the order to get the chat lines from.
+ * @param {Object} options An object of options to configure the request.
+ * @returns {Promise} The order chat's lines.
+ */
+ripe.Ripe.prototype.chatLinesOrderP = function(number, options) {
+    return new Promise((resolve, reject) => {
+        this.chatLinesOrder(number, options, (result, isValid, request) => {
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request, null, result));
+        });
+    });
+};
+
+/**
+ * Returns the number of general chat lines of an order, unrelated to any state.
+ *
+ * @param {Number} number The number of the order to get the number of chat lines from.
+ * @param {Object} options An object of options to configure the request.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.chatLinesCountOrder = function(number, options, callback) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    const url = `${this.url}orders/${number}/chat/lines/count`;
+    options = Object.assign(options, {
+        url: url,
+        method: "GET",
+        auth: true
+    });
+    options = this._build(options);
+    return this._cacheURL(options.url, options, callback);
+};
+
+/**
+ * Returns the number of general chat lines of an order, unrelated to any state.
+ *
+ * @param {Number} number The number of the order to get the number of chat lines from.
+ * @param {Object} options An object of options to configure the request.
+ * @returns {Promise} The number of order chat's lines.
+ */
+ripe.Ripe.prototype.chatLinesCountOrderP = function(number, options) {
+    return new Promise((resolve, reject) => {
+        this.chatLinesCountOrder(number, options, (result, isValid, request) => {
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request, null, result));
+        });
+    });
+};
+
+/**
+ * Adds a new general chat line message to a specific order.
+ *
+ * @param {Number} number The number of the order where the chat line message will be added.
+ * @param {Object} contents The contents of the chat message.
+ * @param {Object} options An object of options to configure the request.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.chatCreateLineOrder = function(number, contents, options, callback) {

```

Comment:
to follow the same format as other methods 

```suggestion
ripe.Ripe.prototype.createChatLineOrder = function(number, contents, options, callback) {

```

Reply:
Following the same formatting as state chat: https://github.com/ripe-tech/ripe-sdk/blob/jc/4702-order-chat-methods/src/js/api/order.js#L914

Reply:
I understand that but I think we follow the same format that is in the majority of the methods to keep consistency